src/handlers/importFileParser/handler.ts

```diff
@@ -0,0 +1,69 @@
+import { S3Event } from "aws-lambda";
+import { S3Client, GetObjectCommand, CopyObjectCommand, DeleteObjectCommand } from '@aws-sdk/client-s3';
+import csv from "csv-parser";
+import { Readable } from "stream";
+
+const s3 = new S3Client();
+
+export const importFileParser = async (event: S3Event) => {
+  if (!event || !event.Records) {
+    throw new Error('Event is not defined');
+  };
+
+  const promises = event.Records.map(async (record) => {
+    try {
+      const bucketName = record.s3.bucket.name;
```

Comment:
Probably it would be better to move it to separate function

Reply:
done